demo/package.json

```diff
@@ -12,7 +12,7 @@
 		"@types/react-dom": "^16.9.0",
 		"prettier": "^1.19.1",
 		"react": "^16.12.0",
-		"react-accessible-dropdown-menu-hook": "link:..",
+		"react-accessible-dropdown-menu-hook": "link:../",
```

Comment:
Is there a reason why this was changed?

Reply:
I've noticed that this changes to this every time that I run `yarn install`, maybe it should be reverted?